bson/src/main/org/bson/types/ObjectId.java

```diff
@@ -255,7 +234,9 @@ public ObjectId(final byte[] bytes) {
      * @param timestamp                   time in seconds
      * @param machineAndProcessIdentifier machine and process identifier
      * @param counter                     incremental value
+     * @deprecated Use {@link #ObjectId(int, int)} instead
      */
+    @Deprecated
```

Comment:
No need to deprecate a package-private method.  It can just be removed if it's no longer required by the driver itself

Reply:
Ack